src/index.css

```diff
@@ -1,19 +1,47 @@
-@import url('https://fonts.googleapis.com/css2?family=Kumbh+Sans:wght@400;700&display=swap');
-:root{
-	--dark-cyan: hsl(185, 75%, 39%);
-	--very-dark-desaturated-blue: hsl(229, 23%, 23%);
-	--dark-grayish-blue: hsl(227, 10%, 46%);
-	--dark-gray: hsl(0, 0%, 59%);
+@import url("https://fonts.googleapis.com/css2?family=Kumbh+Sans:wght@400;700&display=swap");
```

Comment:
Importing fonts in the CSS file is slow. Import them in your `index.html` instead.